packages/plugin-global-user-properties/README.md

```diff
@@ -0,0 +1,66 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# @amplitude/plugin-global-user-properties
+
+Official SDK plugin for adding global user properties to events
+
+## Installation
+
+This package is published on NPM registry and is available to be installed using npm and yarn.
+
+```sh
+# npm
+npm install @amplitude/plugin-global-user-properties
+
+# yarn
+yarn add @amplitude/plugin-global-user-properties
+```
+
+## Usage
+
+This plugin works on top of Amplitude Browser SDK and adds web attribution tracking features to built-in features. To use this plugin, you need to install `@amplitude/plugin-global-user-properties `v2.1.0` or later.
+
+### 1. Import Amplitude packages
+
+* `@amplitude/plugin-global-user-properties`
+
+```typescript
+import * as Amplitude from '@amplitude/analytics-browser'
+import { globalUserPropertiesPlugin } from '@amplitude/plugin-global-user-properties';
+```
+
+### 2. Instantiate page view plugin
+
+The plugin accepts an optional parameter of type `Object` to configure the plugin based on your use case.
+
+```typescript
+const globalUserPropertiesPlugin = globalUserPropertiesPlugin({
+  excludeReferrers: undefined,
+  initialEmptyValue: undefined,
+  resetSessionOnNewCampaign: undefined,
+});
+```
+
+#### Options
+
+|Name|Type|Default|Description|
+|-|-|-|-|
+|`shouldKeepOriginalUserProperties`|`boolean`| `false` | Use this option if you want the user properties to be sent along with the global user properties. This would do nothing for analyses but as of the beta version, global user properties do not appear in Data Governance.|
```

Comment:
I personally find this a bit confusing, especially the part about "This would do nothing for analyses but as of the beta version, global user properties do not appear in Data Governance." Maybe some clarity or further elaboration around relevance would be helpful for users.